### ISSUE TITLE
Missed "Loader" import in MTLLoader

### ISSUE THREAD

**Issue by kirillovdm** [Itee/threejs-full-es6#10]
Hi! I tried to load .mtl in my scene and got the next error:

```
Uncaught ReferenceError: Loader is not defined
    at webpackJsonp../node_modules/threejs-full-es6/builds/Three.es.js.MTLLoader.MaterialCreator.loadTexture (Three.es.js:70426)
```

In this part of code:
```
	loadTexture: function ( url, mapping, onLoad, onProgress, onError ) {

		var texture;
		var loader = Loader.Handlers.get( url );
```

And as i can see you didn't import **Loader** into MTLLoader.js
This is your imports:
```
import { FileLoader } from '../loaders/FileLoader.js'
import { Color } from '../math/Color.js'
import { MeshPhongMaterial } from '../materials/Materials.js'
import { Vector2 } from '../math/Vector2.js'
import { TextureLoader } from '../loaders/TextureLoader.js'
import {
	FrontSide,
	RepeatWrapping
} from '../constants.js'
import { DefaultLoadingManager } from '../loaders/LoadingManager.js'
```

**Comment by kirillovdm** [comment]
Ok. Replacing `Loader.Handlers` to `Loader$1.Handlers` helped resolve the error

**Comment by Itee** [comment]
Thanks for the feedback !

This seems to be an error. And replacing with Loader$1 doesn't looks good to me.
I will take a look about the imports.

**Comment by Itee** [comment]
Ok the import of Loader is now fixed in the next major release v5.0.0 who come tonight !

Could you confirm after the release ?

Best regards
